serverFiles/models/comments.models.js

```diff
@@ -25,4 +25,33 @@ function addCommentByArticleId(article_id, commentData) {
     });
 }
 
-module.exports = addCommentByArticleId;
+function removeCommentById(comment_id) {
+  const regexPattern = /\d/g;
+  const isNumber = regexPattern.test(comment_id);
```

Comment:
This absolutely works however PSQL will automaticaly handle this and reject the promise if the passed ID doesn't match the database schema eg. given a word instead of an integer (which we can use the returned code for to send back the message in the PSQL error handler).